src/www/webpack_electron.js

```diff
@@ -46,6 +46,7 @@ module.exports = makeConfig({
   ],
   extraPlugins: [
     new webpack.DefinePlugin({
+      'outline.WEB_PLATFORM': 'cordova',
```

Comment:
```suggestion
      'outline.WEB_PLATFORM': 'electron',
```
Is copy-paste working in the electron build?

Reply:
No. I wasn't able to build it yet, only if I comment out the other platform.